### ISSUE TITLE
NilEncoding is not available

### ISSUE THREAD

**Issue by Giuliopime** [Alamofire/Alamofire#3707]
* [x] I've read, understood, and done my best to follow the [*CONTRIBUTING guidelines](https://github.com/Alamofire/Alamofire/blob/master/CONTRIBUTING.md).

## What did you do?
Following the guide I tried setting the `nilEncoding` option in the `URLEncodedFormEncoder` with this code:
```swift
        AF.request(
            url,
            parameters: request.parameters,
            encoder: URLEncodedFormParameterEncoder(
                encoder: URLEncodedFormEncoder(nilEncoding: .dropKey),
                destination: .queryString
            )
        )
```

## What did you expect to happen?
I expected it to compile

## What happened instead?
I get the following error:
Incorrect argument label in call (have 'nilEncoding:', expected 'boolEncoding:')

## Alamofire Environment

**Alamofire Version:** 5.6.1
**Dependency Manager:** swift package manager
**Xcode Version:** 14.3
**Swift Version:**5.8
**Platform(s) Running Alamofire:** ios
**macOS Version Running Xcode:** 13.2.1

**Comment by jshier** [comment]
`NilEncoding` is not yet part of a release. You can use `master` until it's released as part of 5.6.5 or 5.7.

**Comment by jshier** [comment]
5.7.0 has been released with `NilEncoding`.
